### ISSUE TITLE
Pixel Metacluster Multiprocess Pool does not close

### ISSUE THREAD

**Issue by srivarra** [angelolab/ark-analysis#767]
*Please refer to our [FAQ](https://ark-analysis.readthedocs.io/en/latest/_rtd/faq.html) and look at our [known issues](https://github.com/angelolab/ark-analysis/issues) before opening a bug report.*

**Describe the bug**

During the multiprocessing step in `pixel_cluster_utils.apply_pixel_meta_cluster_remapping`, occasionly after all FOVs get processed, the pool does not close. This casues the function to hang indefinitly. When restarting the `find_fovs_missing_col` returns an empty list, as there are no more FOVs to process, yet the rest of the function does not get executed. The average channel expression doesn't get re-computed.

**Expected behavior**

The function should continue after the multiprocessing step once it's restarted when all FOVs get processed in the previous execution.

**To Reproduce**
Please either copy/paste or screenshot all of the code you ran which produced the error. In addition, copy the full error message. 

```python
# NOTE: this will only test the else statement in find_fovs_missing_col
fov_list = find_fovs_missing_col(base_dir, pixel_data_dir, 'pixel_meta_cluster_rename')
print("Restarting meta cluster remapping assignment from %s, ""%d fovs left to process" % (fov_list[0], len(fov_list)))

```
The error occurs here when `fov_list = []`, an empty list.

**Comment by alex-l-kong** [comment]
Hopefully addressing this is as simple as including a check for an empty `fov_list`, which should then `return` out of the function.

Although I'm curious why the `print` statement didn't outright fail on an empty `fov_list`, since accessing the `0` index should've caused an `IndexError` to be thrown.

**Comment by srivarra** [comment]
The print statement did indeed fail on the empty `fov_list` and returned out of the function when running the function after the initial execution with the `multiprocessing.Pool` hanging.

**Comment by ngreenwald** [comment]
Lets discuss today at standup

**Comment by alex-l-kong** [comment]
This will be addressed by #771.

**Comment by alex-l-kong** [comment]
Since #771 is merged, this issue should now be fixed.
